### PR TITLE
Fix remove image by Id

### PR DESCRIPTION
**Pull request by jcsirot**
**- What I did**
Fix the `docker app image rm <id>`command with the following behavior
- if the id does not exist a "no such image" error is displayed
- if the id references multiple repositories a "App is referenced in multiple repositories" error is displayed
- if only one App is referenced by this id then this App is removed from the store

**- A picture of a cute animal (not mandatory)**
i6W2VufoExaRznkM

### CODE REVIEW

File: internal/store/bundle.go [hunk 1]
```diff
@@ -112,6 +112,14 @@ func (b *bundleStore) List() ([]reference.Reference, error) {
 
 // Remove removes a bundle from the bundle store.
 func (b *bundleStore) Remove(ref reference.Reference) error {
+	if id, ok := ref.(ID); ok {
+		if len(b.refsMap[id]) == 0 {
+			return fmt.Errorf("no such image %q", reference.FamiliarString(ref))
+		} else if len(b.refsMap[id]) > 1 {
+			return fmt.Errorf("unable to delete %s - App is referenced in multiple repositories", reference.FamiliarString(ref))
```
Comment (rumpl) [inline internal/store/bundle.go]: Use %q here?

Comment (jcsirot) [reply internal/store/bundle.go]: fixed

File: internal/store/bundle.go [hunk 2]
```diff
@@ -112,6 +112,14 @@ func (b *bundleStore) List() ([]reference.Reference, error) {
 
 // Remove removes a bundle from the bundle store.
 func (b *bundleStore) Remove(ref reference.Reference) error {
+	if id, ok := ref.(ID); ok {
+		if len(b.refsMap[id]) == 0 {
+			return fmt.Errorf("no such image %q", reference.FamiliarString(ref))
+		} else if len(b.refsMap[id]) > 1 {
+			return fmt.Errorf("unable to delete %q - App is referenced in multiple repositories", reference.FamiliarString(ref))
```
Comment (rumpl) [inline internal/store/bundle.go]: Should we tell the user which repositories?

Comment (jcsirot) [reply internal/store/bundle.go]: the `docker image rm` command does not do it

**Pull request merged**